docker-compose.yml

```diff
@@ -283,12 +254,9 @@ services:
       - "${OUTBOUND_INTERNAL_PORT}:${OUTBOUND_INTERNAL_PORT}"
 
   broadcast-transformer:
-    container_name: broadcast-transformer
     image: "samagragovernance/broadcast-transformer:${CURRENT_VERSION}"
     restart: always
     env_file: .env
-    networks:
-      - cassandra
     healthcheck:
```

Comment:
Consider removing these since they are not getting utilized anywhere. Or change this to the new aysnc ones.

Reply:
Removed healthchecks from compose as we already have /health